games/space-invaders/Space Invaders [David Winter].json

```diff
@@ -0,0 +1,108 @@
+{
+  "title": "Space Invaders",
+  "authors": [
+    {
+      "id": "david-winter",
+      "name": "David Winter"
+    }
+  ],
+  "release": "1978",
+  "category": "GAME",
```

Comment:
Most of the apps are games, but there are also images (IBM), utilities (keyboard check, opcode test). For the purpose of creating a gallery it would be nice to provide this information.